### PR TITLE
fix(docs): Updated links to API docs

### PR DESCRIPTION
**Pull request by xlisachan**
## Because

- links no longer work as API docs are now in Ecosystem Platform

## This pull request

- updates links to API docs in Ecosystem Platform

_Put an `x` in the boxes that apply_

- [x] My commit is GPG signed.

### CODE REVIEW

File: packages/fxa-auth-server/lib/error.js [hunk 1]
```diff
@@ -267,10 +267,7 @@ AppError.translate = function (request, response) {
       );
     }
   } else if (payload.validation) {
-    if (
-      payload.message &&
-      payload.message.includes('is required')
-    ) {
+    if (payload.message && payload.message.includes('is required')) {
```
Comment (pdehaan) [inline packages/fxa-auth-server/lib/error.js:270]: Nit: use `payload.message?.includes(…)`?

```suggestion
    if (payload.message?.includes('is required')) {
```

Comment (xlisachan) [reply packages/fxa-auth-server/lib/error.js:270]: I feel like we discussed this before.... it probably did not get updated during the joi fiasco. I'll update this, thanks!

File: packages/fxa-auth-server/docs/swagger/auth-server-api.ts [hunk 2]
```diff
@@ -291,7 +291,7 @@ export const AUTH_SERVER_API_DESCRIPTION = {
             "errno": 201,
             "error": "Service Unavailable",
             "message": "Service unavailable",
-            "info": "https://github.com/mozilla/fxa/blob/main/packages/fxa-auth-server/docs/api.md#response-format",
+            "info": "https://mozilla.github.io/ecosystem-platform/api#section/Response-format",
```
Comment (LZoog) [inline packages/fxa-auth-server/docs/swagger/auth-server-api.ts:294]: Could this be worth adding to or creating a `const` somewhere near our API docs so if this ever changes we can update it in one spot?

**Review by LZoog — APPROVED** [review]
Thanks!

File: packages/fxa-auth-server/docs/swagger/auth-server-api.ts (hunk 2 above)
Comment (clouserw) [reply packages/fxa-auth-server/docs/swagger/auth-server-api.ts:294]: In this case it changes so rarely (maybe never!) and it's an easily grepped string I don't think we need to maintain a variable.  Let's land it.

**Pull request merged**